exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillSqlOperator.java

```diff
@@ -147,6 +166,14 @@ public DrillSqlOperatorBuilder setDeterministic(boolean isDeterministic) {
       return this;
     }
 
+    public DrillSqlOperatorBuilder setNiladic(boolean isNiladic) {
+      /*
+       * Set Operand type-checking strategy for an operator which takes no operands
+       */
```

Comment:
Also let's add note here that making function niladic won't allow to select column with function name without table alias. Please also add example:
When querying table t with column session_id:
select session_id from table -> will return generated session_id 
select t1.session_id from table t1 -> will return session_id column value